incubator/zookeeper/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: zookeeper
 home: https://zookeeper.apache.org/
-version: 0.2.2
+version: 0.3.0
```

Comment:
Can we move this chart to stable/ at this point, since it doesn't rely on alpha features anymore. @viglesiasce